### PR TITLE
[GOVCMSD9-716] Lagoon support for PHP8

### PR DESCRIPTION
**Pull request by pandaskii**

### CODE REVIEW

File: drupal/settings/lagoon.settings.php [hunk 1]
```diff
@@ -86,7 +86,8 @@
 
     $redis->connect($redis_host, $redis_port, $redis_timeout);
     $response = $redis->ping();
-    if (strpos($response, 'PONG') === FALSE) {
+
+    if (!$response) {
```
Comment (stooit) [inline drupal/settings/lagoon.settings.php:90]: https://github.com/phpredis/phpredis#ping

> Note: Prior to PhpRedis 5.0.0 this command simply returned the string +PONG.

**Review by steveworley — APPROVED** [review]

**Comment by tobybellwood** [comment]
What's the actual difference here - is it PHP8 or phpredis5 - we have the same logic in our d8 example (albeit not running on PHP8) that we should probably also change then?
https://github.com/amazeeio/drupal-example-simple/blob/9.x-advanced/assets/all.settings.php#L75

**Comment by stooit** [comment]
It's phpredis5, which came with the upstream AIO PHP8.1 image (current 7.x images have phpredis 4.3.0)

Your example project is pinned to 7.4 atm so likely not relevant there until you bump to 8.1 too - can throw a PR your way later if you like

**Comment by tobybellwood** [comment]
Thankfully haven't ported redis across to the php8-based examples yet - will make sure to include this when I do!

**Pull request merged**